crates/holochain_keystore/src/meta_lair_client.rs

```diff
@@ -1,25 +1,16 @@
-use crate::*;
 use holochain_zome_types::Signature;
 use kitsune_p2p_types::dependencies::lair_keystore_api;
 use lair_keystore_api::prelude::*;
-use lair_keystore_api_0_0::actor::Cert as LegacyCert;
-use lair_keystore_api_0_0::actor::CertDigest as LegacyCertDigest;
-use lair_keystore_api_0_0::actor::CertPrivKey as LegacyCertPrivKey;
-use lair_keystore_api_0_0::actor::LairClientApiSender;
 use std::future::Future;
 use std::sync::Arc;
 
 pub use kitsune_p2p_types::dependencies::lair_keystore_api::LairResult;
 
 /// Abstraction around runtime switching/upgrade of lair keystore / client.
-/// Can delete this when we finally delete deprecated legacy lair option.
 #[derive(Clone)]
 pub enum MetaLairClient {
-    /// oldschool deprecated lair keystore client
-    Legacy(KeystoreSender),
-
-    /// new lair keystore api client
-    NewLair(LairClient),
+    /// lair keystore api client
+    Lair(LairClient),
 }
```

Comment:
In light of [this comment](https://github.com/holochain/holochain/pull/1518#issuecomment-1217211018), do we want to be as optimistic as to remove this enum altogether and stick with the `LairClient`? Or would this be the place to enable backward compatibility over various versions?

Reply:
It's a little weird that it's a single variant enum, but the abstraction is useful to tailor it more specifically to holochain, i'd like to keep it for now.

Reply:
https://github.com/holochain/holochain/pull/1529 will change this to a struct so it'll be less weird : )